Protocol.md

```diff
@@ -279,6 +281,58 @@ The recipient:
 
   These script pub key forms include only standard forms accepted by the wider set of deployed Bitcoin clients in the network, which increase the chances of successful propagation to miners.
 
+## Optional features
+
+Optional features consist of "features" which parties can opt-in to using. These messages are not needed to facilitate construction of a DLCs.
+
+### The `close_dlc` Message
+
+This message contains information about mutual close, which allows the counterparty
+to broadcast a mutual closing transaction.
+
+1. type: ? (`close_dlc_v0`)
+2. data:
+   * [`32*byte`:`contract_id`]
+   * [`signature`:`close_signature`]
+   * [`u64`:`offer_payout_satoshis`]
+   * [`u64`:`accept_payout_satoshis`]
+   * [`u64`:`fund_input_serial_id`]
+   * [`u16`:`num_extra_inputs`]
+   * [`num_extra_inputs*input`:`extra_inputs`]
+   * [`input_signatures`:`input_signatures`]
+   * [`u32`:`close_locktime`]
+
+
+`payout_spk` and `payout_serial_id` from `offer_dlc` as well as `payout_spk` and `payout_serial_id` from `accept_dlc` should be used for constructing the close transaction
+
+`fund_input_serial_id` is a randomly chosen number which uniquely identifies the funding output to be spent.
+Inputs in the closing transaction will be sorted by `fund_input_serial_id` and `input_serial_id` in `extra_inputs`.
+
+`extra_inputs` are extra inputs to mutual close to enable the sending party to nullify the transaction by double spending the inputs.
+
+`close_locktime` is the nLockTime to be put in close messages.
+
+#### Requirements
+
+The sender MUST:
+
+  - set `contract_id` from the `sign_dlc` message.
+  - set `close_signature` to a valid signature, using the private key associated with its `funding_pubkey` for the close transaction, as defined in the [transaction specification](Transactions.md#close-transaction).
+  - set `extra_signatures` to contain valid witnesses for every funding input specified by `extra_inputs` and in the same order.
+  - set `close_locktime` to be a UNIX timestamp, or a block height as distinguished [here](https://en.bitcoin.it/wiki/NLockTime).
+  - set `close_locktime` to be less than or equal to current UNIX timestamp or block.
+
+The recipient:
+
+  - if any input in `extra_inputs` is not a BIP141 (Segregated Witness) input.
+    - MAY ignore the message.
```

Comment:
Should require the payouts to be above the dust limit

Reply:
Good catch, added in [a9ce72b](https://github.com/discreetlogcontracts/dlcspecs/pull/170/commits/a9ce72b82514ef34c6333bcd93627f07cddfba23)